<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddWindow</class>
 <widget class="QDialog" name="AddWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>983</width>
    <height>721</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AddInfo</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="0">
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Корпус</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="box_Corpus">
       <item>
        <property name="text">
         <string>Альфа</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Бета</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Вега</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Дельта</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Гамма</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Тип комнаты</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="box_TypeRoom">
       <item>
        <property name="text">
         <string>Стандарт</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Бизнесс-класс</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Первый-класс</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Делюкс</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Фамилия</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="lineEdit_FIO">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Услуги</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QCheckBox" name="checkBox_Pool">
         <property name="text">
          <string>Бассейн</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_Lunch">
         <property name="text">
          <string>Завтрак/Обед/Ужин</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_Fitness">
         <property name="text">
          <string>Фитнесс-зал</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_Spa">
         <property name="text">
          <string>Спа</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_Transport">
         <property name="text">
          <string>Транспорт</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_Wifi">
         <property name="text">
          <string>Wi-fi</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Цена</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLineEdit" name="lineEdit_Cost"/>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>E-mail</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QLineEdit" name="lineEdit_Email"/>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Номер телефона</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QLineEdit" name="lineEdit_PhoneNumber"/>
     </item>
     <item row="9" column="0">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Дата заезда</string>
       </property>
      </widget>
     </item>
     <item row="9" column="1">
      <widget class="QDateEdit" name="dateEdit">
       <property name="maximumDate">
        <date>
         <year>3000</year>
         <month>12</month>
         <day>31</day>
        </date>
       </property>
       <property name="minimumDate">
        <date>
         <year>2024</year>
         <month>1</month>
         <day>1</day>
        </date>
       </property>
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="10" column="0">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>Количество ночей</string>
       </property>
      </widget>
     </item>
     <item row="10" column="1">
      <widget class="QLineEdit" name="lineEdit_NightCount"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>Имя</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="lineEdit_Name"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>Отчество</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="lineEdit_FatherName">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="0">
    <layout class="QFormLayout" name="formLayout_2">
     <item row="0" column="0">
      <widget class="QPushButton" name="pushButton">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 0, 0);
color: rgb(0, 0, 0);</string>
       </property>
       <property name="text">
        <string>Выйти</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="btn_Add">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(0, 170, 0);
selection-background-color: rgb(0, 255, 0);
color: rgb(0, 0, 0);</string>
       </property>
       <property name="text">
        <string>Добавить</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>AddWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>65</x>
     <y>537</y>
    </hint>
    <hint type="destinationlabel">
     <x>5</x>
     <y>519</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
